devops/pipeline/create_containers.yaml

```diff
@@ -39,6 +41,32 @@ parameters:
   type: boolean
   default: false
```

Comment:
Is it by design that Debian 9 is left and not removed?

Reply:
Yes, once we "officially" determine that Debian 10/11 are ready (passed all validation etc.) we will remove support from Debian 9 everywhere in a separate PR.